readthedocs_ext/readthedocs.py

```diff
@@ -50,10 +50,11 @@ def finalize_media(app):
     # Pull project data from conf.py if it exists
     context = app.builder.config.html_context
     STATIC_URL = context.get('STATIC_URL', DEFAULT_STATIC_URL)
-
-    app.builder.script_files.append(
-        '%sjavascript/readthedocs-doc-embed.js' % STATIC_URL
-    )
+    js_file = '{}javascript/readthedocs-doc-embed.js'.format(STATIC_URL)
+    if sphinx.version_info < (1, 8):
+        app.add_javascript(js_file)
+    else:
+        app.add_js_file(js_file)
```

Comment:
Considering my following comment, this may not have a side effect since I suppose that previously and with this change we are adding it at the end of the list. But we should double check, though.

Reply:
It's the same